source/reference/verification.txt

```diff
@@ -16,114 +16,96 @@ Verify Data Transfer
    :depth: 2
    :class: singlecol
 
-When :ref:`mongosync <c2c-mongosync>` has fully committed, verify the 
-successful transfer of your data before you switch your application to using 
-the destination cluster. You can verify your data transfer using document 
-counts, hash comparison, document comparison, or the Migration Verifier.
+When :program:`mongosync` has fully committed, verify the
+successful transfer of your data before you switch your
+application to using the destination cluster.
```

Comment:
Can we also add a note that some of these methods don't require mongosync to commit in order to begin verification?

Reply:
Removed mention of the commit in the intro, will leave the reader to review the table to decide which method to use.